rc/windowing/detection.kak

```diff
@@ -52,7 +52,13 @@ define-command terminal -params 1.. -docstring %{
 
     See also the 'new' command.
 } %{
-    "%opt{windowing_module}-terminal-%opt{windowing_placement}" %arg{@}
+    try %{
+        "%opt{windowing_module}-terminal-%opt{windowing_placement}" %arg{@}
+    } catch %{
+        # falback to window if other placements not supported
+        # maybe we shall let each windowing_module handle windowing_placement instead?
+        "%opt{windowing_module}-terminal-window" %arg{@}
+    }
```

Comment:
I'm not sure, isn't it better to let the user decide explicitly if they want a fallback?

Reply:
you're right, user can provide a command to do it explicitly. Let's close this then.